packages/L10n/src/useI18n.js

```diff
@@ -24,6 +24,7 @@ export const getI18nObject = (locale = "en", locales) => {
   return {
     locale,
     t: _i18n.getFixedT(locale),
+    i18n: _i18n,
```

Comment:
Expose the i18n instance so it can be passed into `<Trans>`

Reply:
While we're at it, can we export `useI18n` from `index.js`, please? So we can import it like:
`import { useI18n } from '@paprika/l10n';`